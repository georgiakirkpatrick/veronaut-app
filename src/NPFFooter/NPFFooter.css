.NPFFooter {
    height: 10vh;
    width: 100%;
    padding: 0 5vw;
    background-color: #2F3B51;
    color: #fff;
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: space-between;
    text-align: match-parent;
    position: absolute;
    bottom: 0;
    left: 0;
    font: 400 16px Poppins;
    border: none;
    border-radius: 0;
}

.NPFFooter button {
    height: 100%;
    background: rgba(255, 255, 255, 0);
    font: 500 16px/20px Poppins, sans-serif;
    color: #fff;
    border: none;
    margin: 0;
}

.NPFFooter__next {
    text-align: right;
}

.NPFFooter__previous {
    text-align: left;
}

.NPFFooter__next,
.NPFFooter__previous {
    padding: 0;
}

.NPFFooter__left-arrow,
.NPFFooter__right-arrow {
    font: 500 16px/20px;
    letter-spacing: 1px;
    font-weight: 100;
}

.NPFFooter__left-arrow {
    margin-right: 3vw;
}

.NPFFooter__right-arrow {
    margin-left: 3vw;
}

@media screen and (min-width: 768px) {
    .NPFFooter {
        padding: 0 7vw;
    }
}

@media screen and (min-width: 1024px) {
    .NPFFooter {
        padding: 0 7vw;
    }
    
    .NPFFooter button {
        margin: 0;
        width: 30%;
        height: 100%;
    }
}

@media screen and (min-width: 1024px) and (min-height: 1366px) {
    .NPFFooter {
        height: 7vh;
        padding: 0 10vw;
    }
}

@media screen and (min-width: 1440px) {
    .NPFFooter {
        padding: 0 15vw;
    }
}

@media screen and (min-width: 2560px) {
    .NPFFooter {
        padding: 0 25vw;
        height: 10vh;
    }
}