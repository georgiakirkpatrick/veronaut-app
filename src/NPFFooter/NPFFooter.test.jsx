import React from 'react'
import { render, screen } from '@testing-library/react'
import { describe, it, expect } from 'vitest'
import NPFFooter from './NPFFooter'

describe('NPFFooter', () => {
  it ('renders without crashing', () => {
    render(<NPFFooter />)
    screen.debug()
  })

  it ('render the UI as expected', () => {
    const tree = render(<NPFFooter />)
    expect(tree).toMatchSnapshot()
  })
})