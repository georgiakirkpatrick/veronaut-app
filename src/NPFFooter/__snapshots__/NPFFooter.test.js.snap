// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NPFFooter render the UI as expected 1`] = `
<footer
  className="NPFFooter"
>
  <button
    className="NPFFooter__previous"
    onClick={[Function]}
  >
    <svg
      aria-hidden="true"
      className="svg-inline--fa fa-angle-left fa-w-8 NPFFooter__left-arrow"
      data-icon="angle-left"
      data-prefix="fas"
      focusable="false"
      role="img"
      style={Object {}}
      viewBox="0 0 256 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
        fill="currentColor"
        style={Object {}}
      />
    </svg>
    <span>
      PREVIOUS
    </span>
  </button>
</footer>
`;
